### ISSUE TITLE
Unsupported operand type error in get_camera_list

### ISSUE THREAD

**Issue by tauren** [unifi-toolbox/unifi-protect-video-downloader#40]
I've used the v1 branch of this project in the past successfully on an older Mac. I'm now trying to use the current recommended docker-based solution on my new M1 Mac. I believe I have things configured properly, as it appears to get the camera list. But it then fails with this error:

```
❯ docker run --volume /Users/tauren/Downloads/LeeSt:/downloads unifitoolbox/protect-archiver download --address=192.168.1.12 --username="unifi-toolbox" --password="*****" --cameras="all" --start="2021-01-07 08:00:00-0800" --end="2021-01-07 08:15:00-0800" --skip-existing-files --touch-files --use-subfolders --download-request-timeout 300 /downloads
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
Getting camera list
Successfully authenticated as user unifi-toolbox
Successfully retrieved data from /api/cameras
Traceback (most recent call last):
  File "/usr/local/bin/protect-archiver", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.8/site-packages/protect_archiver/cli/__init__.py", line 20, in main
    cli.main()
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/protect_archiver/cli/download.py", line 185, in download
    camera_list = client.get_camera_list()
  File "/usr/local/lib/python3.8/site-packages/protect_archiver/client.py", line 271, in get_camera_list
    camera["stats"]["video"]["recordingStart"] / 1000
TypeError: unsupported operand type(s) for /: 'NoneType' and 'int'
```

Running on a CloudKey with 1.1.6 firmware.
Protect Controller 1.13.7 Web UI 1.22.2

**Comment by danielfernau** [comment]
Not sure if that's an architecture problem or just an issue with the code itself. The `TypeError` could theoretically also be caused by an unexpected reply from the Protect system. I'll have a look at it -- thanks for reporting!

**Comment by tauren** [comment]
Perfect, looking forward to your progress! I'd love to get this working on my new Mac.

**Comment by danielfernau** [comment]
Solved a while ago in PR #42

**Comment by adriankeenan** [comment]
Hi, I reproduce this issue when the camera list includes a camera which has never recording anything. `camera["stats"]["video"]["recordingStart"]`  will be `None` in this case but this is assumed to be an `int`.

It appears that you can work around this by enabling then disabling recording (and optionally deleting the recorded clip). This sets the `recordingStart` value which fixes the issue.

**Comment by danielfernau** [comment]
Hi @adriankeenan - thanks for pointing this out. 
I'll try to add some code to handle this scenario a bit better, basically by ignoring all cameras that don't have a valid recordingStart value.

New issue based on your feedback ➡️ #70

I hope to have a little more time this year to maintain the project and add some useful features.

**Comment by danielfernau** [comment]
Should be fixed in latest edge build (via PR #89)
